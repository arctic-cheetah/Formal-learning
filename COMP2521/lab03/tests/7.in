4
9
15
1
10
6
5
12
11
3
13
8
2
7
14
