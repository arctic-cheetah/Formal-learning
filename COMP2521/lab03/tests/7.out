Tree:
    4
   / \
  /   \
 /     \
1       9
 \     / \
  3   /   \
 /   /     \
2   6      15
   / \     /
  5   8   10
     /     \
    7      12
           / \
          /   \
         11   13
                \
                14

#nodes = 15,  #leaves = 5
Infix  : 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 
Prefix : 4 1 3 2 9 6 5 8 7 15 10 12 11 13 14 
Postfix: 2 3 1 5 7 8 6 11 14 13 12 10 15 9 4 
ByLevel: 4 1 9 3 6 15 2 5 8 10 7 12 11 13 14 
